### ISSUE TITLE
Korean/English Toggle Key is not recognized by Osu! mania keymap.

### ISSUE THREAD

**Issue by red-crate** [ppy/osu-stable-issues#773]
There is another key on the Korean keyboard. 
Korean/English Toggle Key.
This key is recognized in the chat inside the game, but not in the game.

some korean keyboards have separate Toggle Key. but in some cases, standard ANSI arrays are adopted.
In this case, it has the same arrangement as a typical English keyboard.
However, in this case, the right ALT key is used as the Korean/English Toggle Key and the right CTRL key is used as the Chinese character Toggle Key.
In other words, right ALT and right CTRL keys cannot be keymap to Osu! mania.

It is a problem that can be solved if the keyboard attribute is changed to English-only each time the game is played.
But it's very tiring and annoying, and most people won't do it.
Therefore, this is causing inconvenience to most Korean players.

Therefore, please fix it so that we can use toggle keys.

**Comment by peppy** [comment]
I don't think we will be able to fix this on stable, but have you tested this on lazer? https://github.com/ppy/osu#releases 

If it doesn't already work there, we may be able to investigate further and provide support for this in the future.

**Comment by red-crate** [comment]
> 
> 
> I don't think we will be able to fix this on stable, but have you tested this on lazer? https://github.com/ppy/osu#releases
> 
> If it doesn't already work there, we may be able to investigate further and provide support for this in the future.

I'm at work now, so I can't check with various keyboards, but there's no right ALT key on the keyboard, only the Korean/English Toggle Key.
At least, the toggle key is not recognized in the latest osu! laser.
